hazelcast/src/main/java/com/hazelcast/client/impl/protocol/ClientProtocolErrorCodes.java

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright (c) 2008-2015, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.client.impl.protocol;
+
+/**
+ * Each exception that are defined in client protocol have unique identifier which are error code.
+ * All error codes defined in protocol are listed in this class.
+ */
+public final class ClientProtocolErrorCodes {
+
+    public static final int UNDEFINED = 0;
```

Comment:
It is error code that we use when exception does not match any of the listed exceptions. I will add the javadoc later.